index.js

```diff
@@ -41,7 +35,7 @@ var re = /require\(['"]lazy-cache['"]\)/;
  */
 
 function isLazy(fp, source) {
-  return (re.test(source) && fp.indexOf('lazy-cache') === -1);
```

Comment:
This was testing to make sure that the source code actually has `lazy-cache` being used in the file being transformed. Otherwise, we just want to skip all the transform logic and return the original source code.

The regex test for `lazy-cache` being required in was the first thing that popped into my head on how to test for that. If you have a better idea, I'm open to that. Rest of the code looks great though and I'm glad you added tests (I wasn't sure how to test webpack loaders, even though they are just functions).